tests/conftest.py

```diff
@@ -334,6 +335,10 @@ def pytest_collection_modifyitems(session, config, items):
                 item.user_properties.append(("squad", squad.capitalize()))
 
     if not (teardown or deploy or (deploy and skip_ocs_deployment)):
+        if ocsci_config.RUN.get("extra_markexpr"):
```

Comment:
We might still have a problem with upgrade scenario when we don't want to have this tests selected.

I am still not sure we want to have this kind of logic if it will be beneficial for us or not. Still trying to go with some pros and cons.